CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -7,28 +7,23 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 ## v3.5.8 (TBD)
 
 ### etcd server
-- Fix [server/embed: fix data race when starting both secure & insecure gRPC servers on the same address](https://github.com/etcd-io/etcd/pull/15517)
+- Fix [Watch response traveling back in time when reconnecting member downloads snapshot from the leader](https://github.com/etcd-io/etcd/pull/15515)
+- Change [http2 frame scheduler to random algorithm](https://github.com/etcd-io/etcd/pull/15452)
+- Fix [race when starting both secure & insecure gRPC servers on the same address](https://github.com/etcd-io/etcd/pull/15517)
 - Add [`etcd --tls-min-version --tls-max-version`](https://github.com/etcd-io/etcd/pull/15483) to enable support for TLS 1.3.
-
-### Package `client/pkg/v3`
-- Fix [aligning zap log timestamp resolution to microseconds](https://github.com/etcd-io/etcd/pull/15240). Etcd now uses zap timestamp format: `2006-01-02T15:04:05.999999Z0700` (microsecond instead of milliseconds precision).
-
-### Package `netutil`
+- Upgrade [go to 1.19.7](https://github.com/etcd-io/etcd/pull/15429)
+- Upgrade [golang.org/x/net to v0.7.0](https://github.com/etcd-io/etcd/pull/15337)
+- Bump bbolt to [v1.3.7](https://github.com/etcd-io/etcd/pull/15222).
 - Fix [consistently format IPv6 addresses for comparison](https://github.com/etcd-io/etcd/pull/15187)
```

Comment:
Note #15187 fixed `pkg/netutil`, and it may be used by etcdserver and client (e.g. etcdutl), so it isn't accurate to get it listed under etcdserver.

Reply:
But, it's not. Only server imports it.

Reply:
etcdutl also depends on it via a long path,

`(*v3Manager) Restore`  ->` srv.VerifyBootstrap() (line 277 in etcdtul/snapshot/v3_snapshot.go)` -> ...... -> `resolveURL (line 91 in pkg/netutil/netutil.go)`

Reply:
For normal etcd user reading changlong information about `netutil` fixes is too obscure. Leaving for now, but we should consider renaming this section.